include/actuator/ActuatorBulk.h

```diff
@@ -0,0 +1,75 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+#ifndef ACTUATORFIELDBULK_H_
+#define ACTUATORFIELDBULK_H_
+
+#include <actuator/ActuatorTypes.h>
+#include <actuator/ActuatorSearch.h>
+#include <stk_mesh/base/BulkData.hpp>
+#include <stk_mesh/base/MetaData.hpp>
```

Comment:
Prefer forward declaration here.

Reply:
ack

Reply:
Sorry. Not sure I understood you. Do you mean to say you don't like forward declaration?

Reply:
@psakievich Note you can use this cute thumbs up emoji in github comments :+1:  or maybe the :ok_hand: for  "acknowledged".

Reply:
👌